src/clib/xtg/cube_export_rmsregular.c

```diff
@@ -89,7 +89,10 @@ cube_export_rmsregular(int nx,
             for (i = 1; i <= nx; i++) {
 
                 ic = x_ijk2ic(i, j, k, nx, ny, nz, 0);
-
+                if (ic < 0) {
+                    throw_exception("Outside boundary in cube_export_rmsregular");
+                    return EXIT_FAILURE;
+                }
```

Comment:
Leaving a dangling file pointer here? Close before leaving?

Reply:
Yes, I have a few of those left, will fix.